opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/TaskRegisterFragmentInteractor.java

```diff
@@ -108,9 +110,10 @@ private String groupedRegisteredStructureTasksSelect(String mainCondition) {
                 SPRAYED_STRUCTURES, tableName, FOR, SPRAYED_STRUCTURES, DBConstants.KEY.BASE_ENTITY_ID));
         structureTasksQueryBuilder.mainCondition(mainCondition);
 
-        return String.format(" SELECT %s.*, SUM(CASE WHEN %s.status='COMPLETED' THEN 1 ELSE 0 END) AS %s , COUNT(%s._id) AS %s FROM ( ",
-                "tasks", "tasks", COMPLETED_TASK_COUNT, "tasks", TASK_COUNT) + structureTasksQueryBuilder +
-                " ) AS tasks GROUP BY tasks.structure_id ";
+        return String.format(" SELECT %s.*, SUM(CASE WHEN %s.status='COMPLETED' THEN 1 ELSE 0 END) AS %s , COUNT(%s._id) AS %s, " +
+                        "GROUP_CONCAT(%s.%s || \"%s\" || %s.%s) AS %s FROM ( ",
+                GROUPED_TASKS, GROUPED_TASKS, COMPLETED_TASK_COUNT, GROUPED_TASKS, TASK_COUNT, GROUPED_TASKS, CODE, HYPHEN, GROUPED_TASKS, BUSINESS_STATUS, GROUPED_STRUCTURE_TASK_CODE_AND_STATUS) + structureTasksQueryBuilder +
```

Comment:
This has too many params , remove the table qualifiers.

This will Improve readability